Extraction fix

louyihua opened the pull request:
When the arguments are passed by parse_arguments, no information is keeped. But only string literals should be extracted.
I've a patch for markey (PR#3, https://github.com/EnTeQuAk/markey/pull/3) to provide the function parse_arguments returning type information as well, so that type information can be examined by caller.

I assume that markey's version is 0.8 when the feature metioned above is added.

louyihua commented:
@sarina @nedbat
This is my new patch that depends on the PR#3 for markey. As long as the PR#3 is accepted, this PR can work properly.

sarina commented:
FYI this is related to bugs uncovered in https://github.com/edx/edx-platform/pull/10416

Commit:
Extraction fix

When the arguments are passed by parse_arguments, no information is keeped. But only string literals should be extracted.
I've a patch for markey to provide a new function named parse_arguments_with_type, so that type information can be examined by caller.

sarina commented on setup.py:
```diff
@@ -38,7 +38,7 @@ def read(*parts):
     'babel>=1.3',
     'django-babel>=0.3.9',
-    'markey>=0.7',
+    'markey>=0.8',
```
@EnTeQuAk this is expecting a version of `markey` that does not yet exist: https://github.com/EnTeQuAk/markey/releases
